mypy.ini

```diff
@@ -0,0 +1,215 @@
+# Global options:
+
+[mypy]
+ignore_missing_imports = True
+python_version = 3.10
+warn_unused_configs = True
+strict_equality = True
+
+# Disallow dynamic typing
+disallow_any_decorated = True
+# TODO(Philipp, 09/23): Remove me!
```

Comment:
Minor: Instead of `disallow_any_explicit = False` we can have a commented out version:
```
# disallow_any_explicit = True
```